From: Roland Dreier <rdreier@cisco.com>
Subject: x86: Remove CPU cache size output for non-Intel too
References: bnc#564618, FATE#306952
Patch-mainline: 2.6.33-rc1
Git-commit: b01c845f0f2e3f9e54e6a78d5d56895f5b95e818

As Dave Jones said about the output in intel_cacheinfo.c: "They
aren't useful, and pollute the dmesg output a lot (especially on
machines with many cores).  Also the same information can be
trivially found out from userspace."

Give the generic display_cacheinfo() function the same treatment.

Signed-off-by: Roland Dreier <rolandd@cisco.com>
Acked-by: Dave Jones <davej@redhat.com>
LKML-Reference: <adaocn6dp99.fsf_-_@roland-alpha.cisco.com>
Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
Automatically created from "patches.arch/x86-Remove-CPU-cache-size-output-for-non-Intel-too.patch" by xen-port-patches.py

--- head-2010-01-18.orig/arch/x86/kernel/cpu/common-xen.c	2009-11-06 10:52:22.000000000 +0100
+++ head-2010-01-18/arch/x86/kernel/cpu/common-xen.c	2010-01-18 17:05:18.000000000 +0100
@@ -419,8 +419,6 @@ void __cpuinit display_cacheinfo(struct 
 
 	if (n >= 0x80000005) {
 		cpuid(0x80000005, &dummy, &ebx, &ecx, &edx);
-		printk(KERN_INFO "CPU: L1 I Cache: %dK (%d bytes/line), D cache %dK (%d bytes/line)\n",
-				edx>>24, edx&0xFF, ecx>>24, ecx&0xFF);
 		c->x86_cache_size = (ecx>>24) + (edx>>24);
 #ifdef CONFIG_X86_64
 		/* On K8 L1 TLB is inclusive, so don't count it */
@@ -450,9 +448,6 @@ void __cpuinit display_cacheinfo(struct 
 #endif
 
 	c->x86_cache_size = l2size;
-
-	printk(KERN_INFO "CPU: L2 Cache: %dK (%d bytes/line)\n",
-			l2size, ecx & 0xFF);
 }
 
 void __cpuinit detect_ht(struct cpuinfo_x86 *c)
